### PR TITLE
Pia 2873 injecting bottom bar camera screen

### PR DESCRIPTION
**Pull request by AdnanYupi**

### CODE REVIEW

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraActivity.java [hunk 1]
```diff
@@ -390,12 +396,29 @@ private void restoreSavedState(@Nullable final Bundle savedInstanceState) {
...
+            adapter.setTitle(isTablet() ? getString(R.string.gc_camera_title) : null);
+
+            adapter.setOnBackButtonClickListener(v -> onBackPressed());
+
+            adapter.setOnHelpButtonClickListener(v -> startHelpActivity());
         }
+
+    }
+
+    private boolean isTablet() {
```
Comment (a-szotyori) [inline capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraActivity.java:420]: [Suggestion] We have a helper method for this called `ContextHelper.isTablet()`. Sorry for not noticing this earler.

**Review by a-szotyori — CHANGES_REQUESTED** [review]
Nice! UI looks great and I have just some small requests.

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/view/CameraBottomNavigationBar.kt [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+interface CameraBottomNavigationBar: InjectedViewAdapter {
+
+    fun setOnBackButtonClickListener(click: View.OnClickListener)
+
+    fun setOnHelpButtonClickListener(click: View.OnClickListener)
+
+    fun setTitle(title: String?)
+}
+
+class CameraBottomNavigationBarAdapter: CameraBottomNavigationBar {
```
[the remark this answers is not in the document]
Comment (AdnanYupi) [reply capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/view/CameraBottomNavigationBar.kt]: Done

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/view/CameraBottomNavigationBar.kt [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import net.gini.android.capture.databinding.GcCameraBottomBarBinding
+import net.gini.android.capture.view.InjectedViewAdapter
+
+interface CameraBottomNavigationBar: InjectedViewAdapter {
+
+    fun setOnBackButtonClickListener(click: View.OnClickListener)
+
+    fun setOnHelpButtonClickListener(click: View.OnClickListener)
+
+    fun setTitle(title: String?)
```
[the remark this answers is not in the document]
Comment (AdnanYupi) [reply capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/view/CameraBottomNavigationBar.kt]: Done

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraActivity.java (hunk 1 above)
Comment (AdnanYupi) [reply capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraActivity.java]: Done

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/view/CameraBottomNavigationBar.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (a-szotyori) [reply capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/view/CameraBottomNavigationBar.kt]: @AdnanYupi FYI: I pushed a commit with updated names:
* adapter: CameraNavigationBarBottomAdapter
* default implementation: DefaultCameraNavigationBarBottomAdapter

I also updated the method and parameter names in GiniCapture.

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**